src/openfl/_internal/renderer/canvas/CanvasBitmap.hx

```diff
@@ -29,14 +29,16 @@ class CanvasBitmap {
 			context.globalAlpha = bitmap.__worldAlpha;
 			var transform = bitmap.__renderTransform;
 			var scrollRect = bitmap.__scrollRect;
+			var scale = renderSession.scale;
+			@:privateAccess var sizeScale = scale / bitmap.__bitmapData.__scale;
```

Comment:
since we're renaming scales to `pixelRatio`/`__pixelRatio`, the `sizeScale` can now be simply called scale, for ez reading

Reply:
I vote for this ;)